gui/layers/_markers_layer.py

```diff
@@ -227,7 +245,7 @@ def scaling(self, scaling: bool) -> None:
 
     def _get_shape(self):
         if len(self.coords) == 0:
-            return np.ones(self.coords.shape, dtype=int)
+            return np.ones(self.coords.shape[1], dtype=int)
```

Comment:
Can you explain what's going on here?

Reply:
This is a bug fix from before. It only applies to when a new layer is created and no coordinates have been passed over. In that case we return a vector of ones with length the dimension of the layer. The dimension of the layer corresponds to the `coords.shape[1]` as `coords` is a 2D array which is NxM where there are N markers over the M dimensions. Does this sound right? It's maybe a little strange, but it prevents things from breaking when there is no data. We might want to revisit / fix, but can do that in a new PR

Reply:
But why `np.ones`? Just so something can be plotted?

Anyway, it might be worth adding a comment,
```
# when no markers given, return (1,) * dim
# we use coords.shape[1] as the dimensionality of the image
```

Reply:
yup will add!